lib/pharos/kubeadm/cluster_config.rb

```diff
@@ -0,0 +1,208 @@
+# frozen_string_literal: true
+
+module Pharos
+  module Kubeadm
+    class ClusterConfig
+      PHAROS_DIR = Pharos::Kubeadm::PHAROS_DIR
+      AUTHENTICATION_TOKEN_WEBHOOK_CONFIG_DIR = '/etc/kubernetes/authentication'
+      AUDIT_CFG_DIR = (PHAROS_DIR + '/audit').freeze
```

Comment:
I think rubocop will complain about .freezing immutables for these

Reply:
I didn't change anything here.. just refactored code to a different class.